advanced/Scripts/webpage.sh

```diff
@@ -58,8 +58,15 @@ delete_setting() {
 }
 
 change_setting() {
-    delete_setting "${1}"
-    add_setting "${1}" "${2}"
+    # Ensure the deleteFTLsetting call succeeds before adding the new setting's value.
+    # On failure, print the error but allow the script to continue.
+    success=true
+    deleteFTLsetting "${1}" || success=false
+    if [ "${success}" == true ]; then
```

Comment:
Why the intermediate global variable?

How about
```suggestion
    if deleteFTLsetting "${1}"; then
```
?

Also, what happens with the error message printed by `sed` in `deleteFTLsetting` ?

Reply:
> Why the intermediate global variable?

Because I'm a shell scripting newbie is why :) Your suggestion works well. The `sed` output was showing up in the startup log. I've refactored to pipe all `sed` output to `/dev/null`

I also noticed I didn't update the correct function before submitting the PR, ooof; sorry. That being said, would make sense to have the same error handling/output discarding for `change_setting()` and `delete_setting()`?